{
  "uid" : "dbaddc04a9248330",
  "name" : "test_get_css_property_footer_license[font-size-16px-\\u0440\\u0430\\u0437\\u043c\\u0435\\u0440\\u0430 \\u0448\\u0440\\u0438\\u0444\\u0442\\u0430-https://pulsewave.ru/auth/signup-chrome]",
  "fullName" : "tests.test_footer.test_footer.TestFooter#test_get_css_property_footer_license",
  "historyId" : "1534dce8b6eabeaf34a788a5dc755a97",
  "time" : {
    "start" : 1714357209629,
    "stop" : 1714357209629,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome-headless-shell=124.0.6367.60)\nStacktrace:\n#0 0x5605688d9e33 <unknown>\n#1 0x5605685c9567 <unknown>\n#2 0x5605685c1879 <unknown>\n#3 0x5605685b1852 <unknown>\n#4 0x5605685b2e44 <unknown>\n#5 0x5605685b1bbe <unknown>\n#6 0x5605685b12e0 <unknown>\n#7 0x5605685b1181 <unknown>\n#8 0x5605685af0ec <unknown>\n#9 0x5605685af60a <unknown>\n#10 0x5605685cbeef <unknown>\n#11 0x560568655695 <unknown>\n#12 0x560568636362 <unknown>\n#13 0x560568654ae2 <unknown>\n#14 0x560568636103 <unknown>\n#15 0x5605686071ba <unknown>\n#16 0x560568607b8e <unknown>\n#17 0x56056889e59b <unknown>\n#18 0x5605688a234b <unknown>\n#19 0x56056888a681 <unknown>\n#20 0x5605688a2eb2 <unknown>\n#21 0x56056886f6af <unknown>\n#22 0x5605688c9428 <unknown>\n#23 0x5605688c9600 <unknown>\n#24 0x5605688d8f84 <unknown>\n#25 0x7f50e6094ac3 <unknown>",
  "statusTrace" : "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"c24191f4d64ebde93d82c89d2d2d7f8f\")>\nurl = 'https://pulsewave.ru/auth/signup'\n\n    @pytest.fixture()\n    def footer_open(driver, url):\n        page = FooterPage(driver)\n>       driver.get(url)\n\ntests/test_footer/conftest.py:8: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:356: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:347: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7f7971c31a30>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_REFUSED\\\\n  ...\\\\n#23 0x5605688c9600 \\\\u003Cunknown>\\\\n#24 0x5605688d8f84 \\\\u003Cunknown>\\\\n#25 0x7f50e6094ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\nE         (Session info: chrome-headless-shell=124.0.6367.60)\nE       Stacktrace:\nE       #0 0x5605688d9e33 <unknown>\nE       #1 0x5605685c9567 <unknown>\nE       #2 0x5605685c1879 <unknown>\nE       #3 0x5605685b1852 <unknown>\nE       #4 0x5605685b2e44 <unknown>\nE       #5 0x5605685b1bbe <unknown>\nE       #6 0x5605685b12e0 <unknown>\nE       #7 0x5605685b1181 <unknown>\nE       #8 0x5605685af0ec <unknown>\nE       #9 0x5605685af60a <unknown>\nE       #10 0x5605685cbeef <unknown>\nE       #11 0x560568655695 <unknown>\nE       #12 0x560568636362 <unknown>\nE       #13 0x560568654ae2 <unknown>\nE       #14 0x560568636103 <unknown>\nE       #15 0x5605686071ba <unknown>\nE       #16 0x560568607b8e <unknown>\nE       #17 0x56056889e59b <unknown>\nE       #18 0x5605688a234b <unknown>\nE       #19 0x56056888a681 <unknown>\nE       #20 0x5605688a2eb2 <unknown>\nE       #21 0x56056886f6af <unknown>\nE       #22 0x5605688c9428 <unknown>\nE       #23 0x5605688c9600 <unknown>\nE       #24 0x5605688d8f84 <unknown>\nE       #25 0x7f50e6094ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:229: WebDriverException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "footer_open",
    "time" : {
      "start" : 1714357209829,
      "stop" : 1714357209981,
      "duration" : 152
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome-headless-shell=124.0.6367.60)\nStacktrace:\n#0 0x5605688d9e33 <unknown>\n#1 0x5605685c9567 <unknown>\n#2 0x5605685c1879 <unknown>\n#3 0x5605685b1852 <unknown>\n#4 0x5605685b2e44 <unknown>\n#5 0x5605685b1bbe <unknown>\n#6 0x5605685b12e0 <unknown>\n#7 0x5605685b1181 <unknown>\n#8 0x5605685af0ec <unknown>\n#9 0x5605685af60a <unknown>\n#10 0x5605685cbeef <unknown>\n#11 0x560568655695 <unknown>\n#12 0x560568636362 <unknown>\n#13 0x560568654ae2 <unknown>\n#14 0x560568636103 <unknown>\n#15 0x5605686071ba <unknown>\n#16 0x560568607b8e <unknown>\n#17 0x56056889e59b <unknown>\n#18 0x5605688a234b <unknown>\n#19 0x56056888a681 <unknown>\n#20 0x5605688a2eb2 <unknown>\n#21 0x56056886f6af <unknown>\n#22 0x5605688c9428 <unknown>\n#23 0x5605688c9600 <unknown>\n#24 0x5605688d8f84 <unknown>\n#25 0x7f50e6094ac3 <unknown>\n\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 917, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/PulseWave/PulseWave/tests/test_footer/conftest.py\", line 8, in footer_open\n    driver.get(url)\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 356, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 347, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  }, {
    "name" : "clear_allure_results_folder",
    "time" : {
      "start" : 1714357159972,
      "stop" : 1714357159973,
      "duration" : 1
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "driver",
    "time" : {
      "start" : 1714357209630,
      "stop" : 1714357209828,
      "duration" : 198
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "_session_faker",
    "time" : {
      "start" : 1714357159955,
      "stop" : 1714357159972,
      "duration" : 17
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1714357210013,
      "stop" : 1714357210070,
      "duration" : 57
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "Тестирование Футера"
  }, {
    "name" : "tag",
    "value" : "regress"
  }, {
    "name" : "parentSuite",
    "value" : "tests.test_footer"
  }, {
    "name" : "suite",
    "value" : "test_footer"
  }, {
    "name" : "subSuite",
    "value" : "TestFooter"
  }, {
    "name" : "host",
    "value" : "fv-az842-111"
  }, {
    "name" : "thread",
    "value" : "2300-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.test_footer.test_footer"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "browser",
    "value" : "'chrome'"
  }, {
    "name" : "css_property",
    "value" : "'font-size'"
  }, {
    "name" : "figma",
    "value" : "'16px'"
  }, {
    "name" : "name",
    "value" : "'размера шрифта'"
  }, {
    "name" : "url",
    "value" : "'https://pulsewave.ru/auth/signup'"
  } ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 2,
        "skipped" : 0,
        "passed" : 11,
        "unknown" : 0,
        "total" : 13
      },
      "items" : [ {
        "uid" : "44842614c3d8b2c0",
        "reportUrl" : "https://Nat754.github.io/PulseWave/8//#testresult/44842614c3d8b2c0",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55e45c2ece33 <unknown>\n#1 0x55e45bfdc567 <unknown>\n#2 0x55e45c02727d <unknown>\n#3 0x55e45c027331 <unknown>\n#4 0x55e45c06a624 <unknown>\n#5 0x55e45c04938d <unknown>\n#6 0x55e45c067ae2 <unknown>\n#7 0x55e45c049103 <unknown>\n#8 0x55e45c01a1ba <unknown>\n#9 0x55e45c01ab8e <unknown>\n#10 0x55e45c2b159b <unknown>\n#11 0x55e45c2b534b <unknown>\n#12 0x55e45c29d681 <unknown>\n#13 0x55e45c2b5eb2 <unknown>\n#14 0x55e45c2826af <unknown>\n#15 0x55e45c2dc428 <unknown>\n#16 0x55e45c2dc600 <unknown>\n#17 0x55e45c2ebf84 <unknown>\n#18 0x7fb0a2e94ac3 <unknown>",
        "time" : {
          "start" : 1714271372560,
          "stop" : 1714271372560,
          "duration" : 0
        }
      }, {
        "uid" : "29e7e620e73f98b9",
        "reportUrl" : "https://Nat754.github.io/PulseWave/7//#testresult/29e7e620e73f98b9",
        "status" : "passed",
        "time" : {
          "start" : 1714184757589,
          "stop" : 1714184757624,
          "duration" : 35
        }
      }, {
        "uid" : "55d86e12115e67c0",
        "reportUrl" : "https://Nat754.github.io/PulseWave/6//#testresult/55d86e12115e67c0",
        "status" : "passed",
        "time" : {
          "start" : 1714148267890,
          "stop" : 1714148267927,
          "duration" : 37
        }
      }, {
        "uid" : "b7524ef0792d49b3",
        "reportUrl" : "https://Nat754.github.io/PulseWave/6//#testresult/b7524ef0792d49b3",
        "status" : "passed",
        "time" : {
          "start" : 1714098314753,
          "stop" : 1714098314787,
          "duration" : 34
        }
      }, {
        "uid" : "a818980cc3fc8c7c",
        "reportUrl" : "https://Nat754.github.io/PulseWave/5//#testresult/a818980cc3fc8c7c",
        "status" : "passed",
        "time" : {
          "start" : 1714014225447,
          "stop" : 1714014225480,
          "duration" : 33
        }
      }, {
        "uid" : "80662c6f66cffcbd",
        "reportUrl" : "https://Nat754.github.io/PulseWave/211//#testresult/80662c6f66cffcbd",
        "status" : "passed",
        "time" : {
          "start" : 1713945708030,
          "stop" : 1713945708062,
          "duration" : 32
        }
      }, {
        "uid" : "6cdf2ddc94230c52",
        "reportUrl" : "https://Nat754.github.io/PulseWave/4//#testresult/6cdf2ddc94230c52",
        "status" : "passed",
        "time" : {
          "start" : 1713925501786,
          "stop" : 1713925501820,
          "duration" : 34
        }
      }, {
        "uid" : "1049ca6ccd764bca",
        "reportUrl" : "https://Nat754.github.io/PulseWave/211//#testresult/1049ca6ccd764bca",
        "status" : "passed",
        "time" : {
          "start" : 1713901333311,
          "stop" : 1713901333346,
          "duration" : 35
        }
      }, {
        "uid" : "c004af7f1bc8c66d",
        "reportUrl" : "https://Nat754.github.io/PulseWave/210//#testresult/c004af7f1bc8c66d",
        "status" : "passed",
        "time" : {
          "start" : 1713885489515,
          "stop" : 1713885489548,
          "duration" : 33
        }
      }, {
        "uid" : "2482bb258420a0d5",
        "reportUrl" : "https://Nat754.github.io/PulseWave/209//#testresult/2482bb258420a0d5",
        "status" : "passed",
        "time" : {
          "start" : 1713883399758,
          "stop" : 1713883399791,
          "duration" : 33
        }
      }, {
        "uid" : "124e01b816f94a90",
        "reportUrl" : "https://Nat754.github.io/PulseWave/209//#testresult/124e01b816f94a90",
        "status" : "passed",
        "time" : {
          "start" : 1713881903716,
          "stop" : 1713881903749,
          "duration" : 33
        }
      }, {
        "uid" : "c93db39cc5904673",
        "reportUrl" : "https://Nat754.github.io/PulseWave/208//#testresult/c93db39cc5904673",
        "status" : "passed",
        "time" : {
          "start" : 1713879079511,
          "stop" : 1713879079546,
          "duration" : 35
        }
      } ]
    },
    "tags" : [ "regress" ]
  },
  "source" : "dbaddc04a9248330.json",
  "parameterValues" : [ "'chrome'", "'font-size'", "'16px'", "'размера шрифта'", "'https://pulsewave.ru/auth/signup'" ]
}